src/main/java/org/opensearch/flowframework/model/WorkflowStepValidator.java

```diff
@@ -78,11 +91,22 @@ public static WorkflowStepValidator parse(XContentParser parser) throws IOExcept
                         requiredPlugins.add(parser.text());
                     }
                     break;
+                case TIMEOUT:
+                    try {
+                        timeout = TimeValue.parseTimeValue(parser.text(), TIMEOUT);
+                    } catch (IllegalArgumentException e) {
+                        logger.error("Failed to parse TIMEOUT value for field [" + fieldName + "]", e);
```

Comment:
Nit: use substitutions instead of concatenation.
```suggestion
                        logger.error("Failed to parse TIMEOUT value for field [{}]", fieldName, e);
```